[2025-06-09T07:40:44.332+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-09T07:40:44.343+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stephen_curry_stats_medallion_etl.ensure_bucket_exists manual__2025-06-09T07:40:43.086198+00:00 [queued]>
[2025-06-09T07:40:44.346+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stephen_curry_stats_medallion_etl.ensure_bucket_exists manual__2025-06-09T07:40:43.086198+00:00 [queued]>
[2025-06-09T07:40:44.346+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-06-09T07:40:44.352+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ensure_bucket_exists> on 2025-06-09 07:40:43.086198+00:00
[2025-06-09T07:40:44.355+0000] {standard_task_runner.py:72} INFO - Started process 192 to run task
[2025-06-09T07:40:44.357+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stephen_curry_stats_medallion_etl', 'ensure_bucket_exists', 'manual__2025-06-09T07:40:43.086198+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/medallion_sales_etl_dag.py', '--cfg-path', '/tmp/tmpzyjf8cym']
[2025-06-09T07:40:44.358+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask ensure_bucket_exists
[2025-06-09T07:40:44.387+0000] {task_command.py:467} INFO - Running <TaskInstance: stephen_curry_stats_medallion_etl.ensure_bucket_exists manual__2025-06-09T07:40:43.086198+00:00 [running]> on host 7f4f4ec1a8c4
[2025-06-09T07:40:44.435+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stephen_curry_stats_medallion_etl' AIRFLOW_CTX_TASK_ID='ensure_bucket_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T07:40:43.086198+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T07:40:43.086198+00:00'
[2025-06-09T07:40:44.436+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-09T07:40:44.453+0000] {logging_mixin.py:190} INFO - Bucket 'mybucket' already exists.
[2025-06-09T07:40:44.454+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-06-09T07:40:44.469+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-09T07:40:44.469+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stephen_curry_stats_medallion_etl, task_id=ensure_bucket_exists, run_id=manual__2025-06-09T07:40:43.086198+00:00, execution_date=20250609T074043, start_date=20250609T074044, end_date=20250609T074044
[2025-06-09T07:40:44.498+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-09T07:40:44.514+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-09T07:40:44.515+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
